### DISCUSSION TITLE
Redox OS support

### DISCUSSION THREAD

**Discussion by iaxat** [zed-industries/zed#25444] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

Right now, Zed is planned for mainstream platforms. 
Can this be ported to Redox OS?

**Comment by maxdeviant** [comment]
We are not looking to port Zed to RedoxOS at this time.

**Comment by ActuallyHappening** [comment]
Would be great, just saying

**Comment by ActuallyHappening** [comment]
I really like Zed as an idea (Rust modern editor) and in its implementation, I know much of the docs are autogenerated which is a really good design decision imho, although I haven't ever contributed or peeked at the source too much. I understand it uses its own rendering engine to keep quick startup times, I know this because I was hoping to integrate some of its functionality in a `bevy` app that uses `webgpu` under the hood but realised thats not feasible.
Not to propose a bad engineering option, but one possible direction for Zed would be to keep its custom rendering engine's frontend API similar enough to `webgpu` that it gets the support of `webgpu` "for free" on platforms that don't have its own custom implementation. I'm no expert in this area though. This would "automatically" give it support on RedoxOS and on browsers supporting WebGPU (obviously that would be a stretch), plus I could then use it, with some heavy compat layers I'm sure, in my `bevy` applications
